content/projects/projectomes.html

```diff
@@ -61,6 +61,12 @@ <h2>M2G: Reliable Human Connectomes At Scale</h2>
     </select>
 </div>
 
+<div class="detail-wrap">
+    <div class="intro clearfix">
+        <h3>About</h3>
+        <p>This page hosts human connectomes produced with <a href="https://github.com/neurodata/ndmg">Neurodata's MRI Graphs pipeline (ndmg)</a>. Data are from a multitude of labs, and begin as either fMRI or dMRI. The data are <a href="https://www.biorxiv.org/content/early/2018/04/24/188706">run through ndmg,</a> and after processing, we provide summary statistics, QA, and the connectomes themselves.</p>
+    </div>
+
 <div class="detail-wrap">
```

Comment:
we don't want multiple levels of "detail-wrap" class -- it causes indentation each time (delete this second instance of this)